scripts/build.sh

```diff
@@ -149,7 +181,7 @@ export INSTALL_ROOT
 # make sure we find our own packages first (e.g., libgit2 only uses pkg_config to find libssh2)
```

Comment:
Wtf, You can't do comments on the bits between the hunks. Why github.

Anyway, shouldn't the install-root-bits end up being prefixed to? otherwise all those bits will conflict and overwrite each other.

Reply:
Yes, I think that makes sense. Will add.